src/utils/querybuilder.util.js

```diff
@@ -5,8 +5,8 @@
  * @return a mongo regex query
  */
 let stringQueryBuilder = function (target) {
-  let t2 = target.replace(/[\\(\\)\\-\\_\\+\\=\\/\\.]/g, '\\$&');
```

Comment:
linter

Reply:
> I ran locally, ran test. It works as expected. It's hard to digest what the changes are on the queryBuilder file.
> 
> I think the main thing to consider is indexing the `meta.lastUpdated`, and whether you want that to live in the code (preferred) or in mongo. Otherwise, it looks to behave as expected.

We have CI setup on this repo so all the tests automatically run on Github too.

Reply:
Added index on meta.lastUpdated.  Thanks for the suggestion